.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,48 @@
+<!-- Before filing an issue, please search the existing issues (both open and closed) to see if your report might be duplicate -->
+<!-- Please don't file an issue when you have a support question, send support questions to the mailinglist (https://www.powerdns.com/mailing-lists.html) -->
```

Comment:
maybe mention IRC here, i.e. link to https://www.powerdns.com/opensource.html instead?